java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -183,4 +184,31 @@ CompletableFuture<Double> zaddIncr(
      * }</pre>
      */
     CompletableFuture<Long> zcard(String key);
+
+    /**
+     * Returns the number of members in the sorted set stored at <code>key</code> with scores between
+     * <code>minScore</code> and <code>maxScore</code>.
+     *
+     * @see <a href="https://redis.io/commands/zcount/">redis.io</a> for more details.
+     * @param key The key of the sorted set.
+     * @param minScore The minimum score to count from. Can be an implementation of InfBound
+     *     representing positive/negative infinity, or ScoreBoundary representing a specific score and
+     *     inclusivity.
```

Comment:
Might be a bit clearer, same suggestion for the other docs
```suggestion
     *     representing positive/negative infinity, or ScoreBoundary representing a specific inclusive or exclusive score.
```